splunk_eventgen/splunk_app/README/eventgen.conf.spec

```diff
@@ -265,6 +265,12 @@ mode = sample | replay
     * Default is sample, which will generate count (+/- rating) events every configured interval
     * Replay will instead read the file and leak out events, replacing timestamps,
 
+splitSample = <integer>
+    * only works with mode sample
```

Comment:
@lephino this is the line I mentioned earlier. Can you update this to include a brief mention of replay mode usage?